### PR TITLE
Update unsorted.m3u

### PR DESCRIPTION
**Pull request by freearhey**
Removed broken links.

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
I'll take a look in a few hours.

**Comment by LaneSh4d0w** [comment]
For the TVMak Albanian links,

Move server to https://t1.tvmak.com/
Add token like this ?HEQu1LZGBt
Add referer https://tvmak.com/
Add user-agent Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:93.0) Gecko/20100101 Firefox/93.0

**Review by BellezaEmporium — COMMENTED** [review]
LGTM for the rest.

File: channels/unsorted.m3u [hunk 1]
```diff
@@ -353,16 +275,10 @@ http://62.113.210.250:1935/medienasa-live/_definst_/mp4:ran1_high/playlist.m3u8
...
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Real Family TV
-https://simultv.s.llnwi.net/o059/RealFamilyTV/interlink.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Red Bull TV
 https://rbmn-live.akamaized.net/hls/live/622817/BoRB-US/master_928.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Red TV
 https://live.rednet.rs/providus/redtv_multi_hq/index.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Red TV
-https://t2.tvmak.com/tv/RTVRED.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",TV Marajoara
-http://video01.kshost.com.br/tv31966/tv31966/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/unsorted.m3u:365]: Went back online to https://video01.kshost.com.br:4443/tv31966/tv31966/playlist.m3u8

**Comment by freearhey** [comment]
I moved all the links from [tvmak.com](https://tvmak.com/) to a separate file `al_tvmak.com`.

**Review by BellezaEmporium — APPROVED** [review]

**Review by sguinetti — APPROVED** [review]

**Pull request merged**